### ISSUE TITLE
Fatal errors in packaged build

### ISSUE THREAD

**Issue by AndyPants550** [ASimpson87/UnrealMosquitto4.21#1]
Thanks this helps a lot.   
But I'm unable to use it in a packaged build.

When i package the project and try to play it I get immediate fatal errors

![CP4](https://user-images.githubusercontent.com/39293865/55460396-2dc76680-55f2-11e9-8e14-d86617b5fd1f.JPG)

**Comment by amar0k** [comment]
I guess that's because it is using editor only functions i will have to refactor them out.

**Comment by AndyPants550** [comment]
Is that possible...to refactor them out?

**Comment by amar0k** [comment]
Yep , I just need to find the time at the moment. Otherwise try package it as development that might work.

**Comment by AndyPants550** [comment]
Hi, I'm not sure development build is gonna help me much, I would like to use it for a final packaged product version.
Hope you find time somewhere to help out.

**Comment by venkyxander** [comment]
Hi,
I am facing the same problem. Has the issue been fixed?

**Comment by PeterPansen** [comment]
Same here. Cannot package project. Error:

`UATHelper: Packaging (HTML5):   Error: Couldn't find parent type for 'K2Node_GetSequenceBinding' named 'UK2Node' in current module or any other module parsed so far.`

Sorry to bother you. Everything else works flawlessly, but without packaging I'm stuck
